### PR TITLE
Fix #1367: Image picker enhancements

### LINKED ISSUE #1367: Image picker enhancements
Site menu interface should match the way the OneDrive section displays folders. When you select the OneDrive section, folders are displayed using the yellow folder icon, but when you select the Site section, only a grey box is displayed.

Upload doesn't prompt with browser file picker when selected. When you click on Upload using the OOTB image picker, it displays a file picker without the user having to click the Choose file link first.
From a link needs the instruction text updating, and doesn't have any placeholder text. OOTB there is "https://" as placeholder text, and different instructions are provided.

Selected text doesn't match site theme colour (our theme in the example screenshots uses red):

### PR DESCRIPTION
**Pull request by Ateina**
| Q               | A
| --------------- | ---
| Bug fix?        | [X]
| New feature?    | [ ]
| New sample?      | [ ]
| Related issues?  | fixes #1367 

#### What's in this Pull Request?
PR contains changes to the FilePicker control to match OOTB Sharepoint File Picker

- Added placeholder to the 'From a link' tab

image

- Updated layout for 'Site' tab

Before:
image

PR:

### CODE REVIEW

File: src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss [hunk 1]
```diff
@@ -2,95 +2,55 @@
...
-    .ms-List-surface {
-      position: relative;
-    }
-  }
+.documentLibraryBrowserContainer {
+  padding: 0;
 }
 
 .filePickerFolderCardTile {
+  max-width: 176px;
```
Comment (michaelmaillot) [inline src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss:10]: From what I tested, this property is not mandatory, as it's already calculated from the whole div.

File: src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss [hunk 2]
```diff
@@ -2,95 +2,55 @@
...
+      text-decoration: underline;
+    }
   }
 
-  &:active {
-    color: "[theme:black, default:#{$ms-color-black}]";
-    background-color: "[theme:neutralLighterAlt, default:#{$ms-color-neutralLighterAlt}]";
+  .filePickerFolderCardImage {
+    position: relative;
+    width: 112px;
```
Comment (michaelmaillot) [inline src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss:26]: For width values, I suggest to indicate viewport unit in order to be more responsive:

```suggestion
    width: 7vw;
```

File: src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss [hunk 3]
```diff
@@ -2,95 +2,55 @@
...
+      }
+    }
   }
-}
 
-.documentLibraryBrowserContainer {
-  padding: 0;
+  .filePickerFolderCardTitle {
+    cursor: pointer;
+    font-size: 14px;
```
Comment (michaelmaillot) [inline src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss:49]: As we're using Fluent UI Sass references, maybe we could use appropriate variables if available, don't you think?

```suggestion
    font-size: $ms-font-size-14;
```

**Review by michaelmaillot — CHANGES_REQUESTED** [review]
Thanks for the PR @Ateina!

I've added few things to update. Let me know if something's not right from your side 🙂

**Comment by Ateina** [comment]
@michaelmaillot Thank you for review, I'm working on update for more responsive layout :)

File: src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss (hunk 2 above)
Comment (Ateina) [reply src/controls/filePicker/controls/DocumentLibraryBrowser/DocumentLibraryBrowser.module.scss]: Updated styles, removed pixel values

**Comment by michaelmaillot** [comment]
Merged manually, thank you!

**Pull request merged**